### DISCUSSION TITLE
running HALFpipe with data already preprocessed in fMRIPrep

### DISCUSSION THREAD

**Discussion by avannucci** [HALFpipe/HALFpipe#163] [Q&A] [answered]
Hi all, I have just finished preprocessing some task-based fMRI data with fMRIPrep when I came across HALFpipe. Is it possible to use HALFpape if you already have the data preprocessed? how would I skip that step and start with post-fMRIPrep steps?

**Comment by HippocampusGirl** [comment, marked as answer]
Dear Anna,
HALFpipe currently does not support importing from fMRIPrep. I have added this as a feature request to the issues tracker here:
- https://github.com/HALFpipe/HALFpipe/issues/169

If you would like to stay notified, you can subscribe to the issue.
